R/steps-rcmdcheck.R

```diff
@@ -71,15 +73,19 @@ RCMDcheck <- R6Class(
 #'   Should warnings be treated as errors? Default: `TRUE`.
 #' @param notes_are_errors `[flag]`\cr
 #'   Should notes be treated as errors? Default: `FALSE`.
-#' @param args `[character]`\cr
+#' @param check_args `[character]`\cr
 #'   Passed to `[rcmdcheck::rcmdcheck()]` (after splitting at spaces), default:
 #'   `"--no-manual --as-cran"`.
+#' @param build_args `[character]`\cr
+#'   Passed to `[rcmdcheck::rcmdcheck()]`
 #' @export
 step_rcmdcheck <- function(warnings_are_errors = TRUE, notes_are_errors = FALSE,
-                           args = "--no-manual --as-cran") {
+                           check_args = "--no-manual --as-cran",
+                           build_args = "--no-multiarch") {
   RCMDcheck$new(
     warnings_are_errors = warnings_are_errors,
     notes_are_errors = notes_are_errors,
-    args = strsplit(args, "[[:blank:]]+")[[1]]
+    check_args = strsplit(check_args, "[[:blank:]]+")[[1]],
```

Comment:
Unrelated to this PR: I no longer like the `strsplit()` here, because this means that we can't pass arguments that contain spaces at all. Maybe `unlist(strsplit(...))` ?